.deepsource.toml

```diff
@@ -0,0 +1,10 @@
+version = 1
+
+test_patterns = ["test/**"]
+
+[[analyzers]]
+name = "go"
+enabled = true
+
+  [analyzers.meta]
+  import_paths = ["github.com/HarshCasper/fyne"]
```

Comment:
I don't know if we want to use deepsource on every commit - but if we did this line would be wrong.

Reply:
I will update this up if there are plans to integrate DeepSource.